recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "1.4.0" %}
-{% set eccodes_version = "2.23.0" %}
+{% set version = "1.4.2" %}
+{% set eccodes_version = "2.26.0" %}
 
 package:
   name: python-eccodes
   version: {{ version }}
 
 source:
-  url: https://github.com/ecmwf/eccodes-python/archive/{{ version }}.tar.gz
-  sha256: 625055bc9efba37d3564562c17779814fa3cc9f6e5ebe0115bf3d78edcc7d503
+  url: https://files.pythonhosted.org/packages/55/d1/511defc811c6ce7cbb0df79969b174fe97ccf7c305380f8842c1311cbbdb/eccodes-{{ version }}.tar.gz
```

Comment:
I don't think https://github.com/ecmwf/eccodes-python/archive/ is active

I got the new url form https://pypi.org/project/eccodes/1.4.2/#files